Models/Text/GPT2/GPT2.swift

```
@@ -155,15 +154,13 @@ public class GPT2 {
             randomCategorialLogits: logits.squeezingShape(at: 1),
             sampleCount: 1)
 
-        guard let id = Int32(seed[0][0]) else {
-          throw GPT2Error.invalidSeed(seed: seed)
-        }
+        let id = Int32(seed[0][0])!
         if id == Int32(endOfTextId) {
             // Replace with newline.
             return "\r\n"
         }
         if let token: String = bpe.vocabulary.token(forId: Int(id)) {
-            let decodedToken = BytePairEncoder.decode(token: token)
+            let decodedToken = try BytePairEncoder.decode(token: token)
```

Should this be `try!` or doesn't `decodedToken` become optional?

---

`decode()` does not return an optional and if it fails we just want to propagate the error, so neither `try!` nor `try?` make sense here.